CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - `datadogexporter`: Add insecure_skip_verify flag to configuration (#7422)
 - `coralogixexporter`: Update readme (#7785)
 - `awscloudwatchlogsexporter`: Remove name from aws cloudwatch logs exporter (#7554)
+- `signalfxexporter`: Updated documentation in the SignalFX exporter for translation rules (#7875)
```

Comment:
I think we don't need a change log line for comment fixes

Reply:
Removed